### PR TITLE
Add video data loader

### PR DESCRIPTION
**Pull request by iejMac**

### CODE REVIEW

**Comment by iejMac** [comment]
For now I'm just adding the webdataset one, if needed I can also add the file one

**Comment by rom1504** [comment]
I think either we document it and expose it properly, either we make it an example

Having it as an example for now could be fine

**Comment by rom1504** [comment]
What's the speed

File: examples/test.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import time
+from data_loader import WebdatasetReader
+
+TARS = "pipe: aws s3 cp s3://s-laion/acav100m/mp4_acav100m/{00000..00000}.tar -"
```
Comment (rom1504) [inline examples/test.py:4]: webdataset parallelism works at the tar level, so you need to pass at least num_prepro_workers tars

File: examples/test.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import time
```
Comment (rom1504) [inline examples/test.py:1]: what about moving this directly into the example

**Comment by iejMac** [comment]
Screen Shot 2022-12-25 at 6 27 19 PM

cpus running pretty well, it gets like 60 vid/s

**Comment by rom1504** [comment]
Lgtm

**Pull request merged**